### ISSUE TITLE
MISP / TheHive feeder broken?

### ISSUE THREAD

**Issue by xme** [CIRCL/AIL-framework#518]
I just restarted my AIL env and it fetches some updates...
MISP_The_Hive_feeder.py is broken:

```
root@ail:/opt/AIL-framework/bin# ./MISP_The_Hive_feeder.py
Misp keys not present
Connected to The HIVE: https://xxxxxxxxx
Traceback (most recent call last):
  File "./MISP_The_Hive_feeder.py", line 225, in <module>
    feeder(message)
  File "./MISP_The_Hive_feeder.py", line 118, in feeder
    create_the_hive_alert(source, path, tag)
NameError: name 'path' is not defined
root@ail:/opt/AIL-framework/bin# 
```

Or I'm missing something?

**Comment by Terrtia** [comment]
Hi @xme !
Thanks for the report !

Fixed with 97905367b64cdcf866d7cdeeb7cad730e9791c69 and 3295b00c69748956729d6d69522bdb843586b37f

**Comment by xme** [comment]
Restarted, git pull, all good now! Tx!
